From 100c7e7c90f5810723c46dbf51b79d76c7855241 Mon Sep 17 00:00:00 2001
From: Steven Barrett <steven@liquorix.net>
Date: Wed, 22 Jan 2020 14:57:10 -0600
Subject: [PATCH 14/15] staging: exfat: Rename "EXFAT_FS" to "EXFAT_FS_SAMSUNG"

Both EXFAT_FS already in the kernel and EXFAT_FS from Samsung both use
the same Kconfig to enable each other.  Lets rename Samsung's
implementation to EXFAT_FS_SAMSUNG in the short term so we don't get
errors from Kconfig itself.
---
 drivers/staging/Makefile       |  2 +-
 drivers/staging/exfat/Kconfig  | 16 ++++++++--------
 drivers/staging/exfat/Makefile |  2 +-
 3 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index a90f9b308c8d..66d8572e914b 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -51,5 +51,5 @@ obj-$(CONFIG_KPC2000)		+= kpc2000/
 obj-$(CONFIG_ISDN_CAPI)		+= isdn/
 obj-$(CONFIG_UWB)		+= uwb/
 obj-$(CONFIG_USB_WUSB)		+= wusbcore/
-obj-$(CONFIG_EXFAT_FS)		+= exfat/
+obj-$(CONFIG_EXFAT_FS_SAMSUNG)		+= exfat/
 obj-$(CONFIG_QLGE)		+= qlge/
diff --git a/drivers/staging/exfat/Kconfig b/drivers/staging/exfat/Kconfig
index 5c1c597e89d7..41bffe4a53c8 100644
--- a/drivers/staging/exfat/Kconfig
+++ b/drivers/staging/exfat/Kconfig
@@ -1,5 +1,5 @@
 # SPDX-License-Identifier: GPL-2.0
-config EXFAT_FS
+config EXFAT_FS_SAMSUNG
 	tristate "exFAT fs support"
 	depends on BLOCK && !EXFAT_FS
 	select NLS
@@ -8,7 +8,7 @@ config EXFAT_FS
 
 config EXFAT_DONT_MOUNT_VFAT
 	bool "Prohibit mounting of fat/vfat filesystems by exFAT"
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	default y
 	help
 	  By default, the exFAT driver will only mount exFAT filesystems, and refuse
@@ -17,34 +17,34 @@ config EXFAT_DONT_MOUNT_VFAT
 
 config EXFAT_DISCARD
 	bool "enable discard support"
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	default y
 
 config EXFAT_DELAYED_SYNC
 	bool "enable delayed sync"
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	default n
 
 config EXFAT_KERNEL_DEBUG
 	bool "enable kernel debug features via ioctl"
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	default n
 
 config EXFAT_DEBUG_MSG
 	bool "print debug messages"
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	default n
 
 config EXFAT_DEFAULT_CODEPAGE
 	int "Default codepage for exFAT"
 	default 437
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	help
 	  This option should be set to the codepage of your exFAT filesystems.
 
 config EXFAT_DEFAULT_IOCHARSET
 	string "Default iocharset for exFAT"
 	default "utf8"
-	depends on EXFAT_FS
+	depends on EXFAT_FS_SAMSUNG
 	help
 	  Set this to the default input/output character set you'd like exFAT to use.
diff --git a/drivers/staging/exfat/Makefile b/drivers/staging/exfat/Makefile
index 6c90aec83feb..e498c4a0ede0 100644
--- a/drivers/staging/exfat/Makefile
+++ b/drivers/staging/exfat/Makefile
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
-obj-$(CONFIG_EXFAT_FS) += exfat.o
+obj-$(CONFIG_EXFAT_FS_SAMSUNG) += exfat.o
 
 exfat-y :=	exfat_core.o	\
 		exfat_super.o	\
-- 
2.25.0.2.g232378479e.dirty

